css/components/page-sections.css

```diff
@@ -31,36 +31,36 @@
   --color-page-section-text-color: var(--color-page-section-background-color-6-text-color);
 }
 
-.lgd-page-section--padding-top-small {
-  padding-top: var(--spacing-small);
+.lgd-page-section--padding-block-start-small {
```

Comment:
Did you mean to update these class names, and if so are they updated in the templates too?

Reply:
Oooh, good find. Let's not update the class names (and maybe in the next phase of this, we can add the new class names into the template as well as the current ones) and in a future phase, remove the current ones.